### ISSUE TITLE
Import from GLB format fails with System.NotImplementedException

### ISSUE THREAD

**Issue by trejkaz** [vrm-c/UniVRM#424]
In attempting to work around the apparent issues with UniVRM dealing with my model imported from FBX, I attempted to import via GLB format instead.

**To Reproduce**

1. New Unity Project
2. Drag in v0.55
3. Unpack this: [ApiMikuGlb.zip](https://github.com/vrm-c/UniVRM/files/4737845/ApiMikuGlb.zip)

**Result:**

```lang-none
import error: C:/Users/Trejkaz/Documents/Unity/UniVRM Normals Issue Take 2/Assets/ApiMiku.glb
UnityEngine.Debug:LogErrorFormat(String, Object[])
UniGLTF.<>c__DisplayClass2_0:<ImportDelayed>b__0() (at Assets/VRM/UniGLTF/Editor/gltfAssetPostprocessor.cs:79)
UnityEditor.EditorApplication:Internal_CallDelayFunctions() (at C:/buildslave/unity/build/Editor/Mono/EditorApplication.cs:209)
```

```lang-none
System.NotImplementedException: different targets: UniGLTF.glTFPrimitives with System.Collections.Generic.List`1[UniGLTF.gltfMorphTarget]
  at DepthFirstScheduler.Schedulable`1[T].ExecuteAll () [0x0003e] in C:\Users\Trejkaz\Documents\Unity\UniVRM Normals Issue Take 2\Assets\VRM\DepthFirstScheduler\Schedulable.cs:119 
  at UniGLTF.ImporterContext.Load () [0x00008] in C:\Users\Trejkaz\Documents\Unity\UniVRM Normals Issue Take 2\Assets\VRM\UniGLTF\Scripts\IO\ImporterContext.cs:438 
  at UniGLTF.gltfAssetPostprocessor+<>c__DisplayClass2_0.<ImportDelayed>b__0 () [0x00002] in C:\Users\Trejkaz\Documents\Unity\UniVRM Normals Issue Take 2\Assets\VRM\UniGLTF\Editor\gltfAssetPostprocessor.cs:65 
UnityEngine.Debug:LogErrorFormat(String, Object[])
UniGLTF.<>c__DisplayClass2_0:<ImportDelayed>b__0() (at Assets/VRM/UniGLTF/Editor/gltfAssetPostprocessor.cs:80)
UnityEditor.EditorApplication:Internal_CallDelayFunctions() (at C:/buildslave/unity/build/Editor/Mono/EditorApplication.cs:209)
```

**Comment by PoChang007** [comment]
Hi, thanks for let us know.
This problem has been fixed in #380. We have tried to import your glb model with development version and it looks fine, so now it is only effective when you clone this repo. This fix will be included in v0.56 release.

**Comment by hiroj** [comment]
fixed in v0.56.2.

**Comment by trejkaz** [comment]
Tried in v0.56.3 with my real model and that failed with new issues. I split those to new tickets:

- #476 - Textures containing Unicode character escapes get written to the wrong files, which aborts the import.
- #477 - System.NullReferenceException from DepthFirstScheduler/Schedulable.cs
